### DISCUSSION TITLE
[Bug]: When trrying to compile rak11310 it errors out.

### DISCUSSION THREAD

**Discussion by Donnie650** [meshtastic/firmware#4020] [Troubleshooting]
### Category

Other

### Hardware

Rak11310

### Firmware Version

2.3.11.2740a56

### Description

When attempting to compile I get this error before it finishes configuration at startup.

### Relevant log output

```Shell
VCSBaseException: VCS: Could not process command ['git', 'clone', '--recursive', '--depth', '1', '--branch', '3.7.2', 'https://github.com/earlephilhower/arduino-pico.git', 'C:\\Users\\User\\.platformio\\.cache\\tmp\\pkg-installing-kksuokzw']
```

**Comment by thebentern** [comment]
The pico toolchain is very big and sometimes the long subdirectory paths break windows.

**Comment by Donnie650** [comment]
Is there anything I can do then on my end to fix? I am using windows 10.

**Reply by fifieldt** [reply]
Some people have had success with the Windows Subsystem for Linux
